R/meta_sl_example.R

```diff
@@ -87,12 +93,14 @@ meta_sl_example <- function() {
     metalite::define_parameter(
       name = "disposition",
       var = "EOSSTT",
-      label = "Trial Disposition"
+      label = "Trial Disposition",
+      varlower = "DCSREAS"
```

Comment:
Hi @fukuhiro2023 , very nice work! Shall we name `varlower` to `var_lower`, following the snake case?

Reply:
Sure, updated.